health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignSummary.js

```
@@ -285,6 +196,133 @@ const CampaignSummary = (props) => {
+      return productVariants.map((variant, key) => ({
+        key: key + 1,
+        count: 1,
+        value: variant.productVariantId,
+        name: variant.name,
+      }));
+    };
+  
+    const parseConditionAndCreateRules = (condition, ruleKey, products) => {
+      // const conditionParts = condition.split("and").map((part) => part.trim());
+      // let attributes = [];
+      let attributes = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'attributes' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)